src/Core/Case/CaseTree.idr

```diff
@@ -207,11 +208,11 @@ mutual
 
   insertCaseAltNames : SizeOf outer ->
                        SizeOf ns ->
-                       CaseAlt (outer ++ inner) ->
-                       CaseAlt (outer ++ (ns ++ inner))
+                       CaseAlt (outer +%+ inner) ->
+                       CaseAlt (outer +%+ (ns +%+ inner))
   insertCaseAltNames p q (ConCase x tag args ct)
       = ConCase x tag args
-           (rewrite appendAssociative args outer (ns ++ inner) in
+           (rewrite appendAssociative args outer (ns +%+ inner) in
```

Comment:
`appendAssociative` is still List-specific

Reply:
Yep, current step is only renaming to mark existing usages of `List` in context of `Scope`. No structural changes for current step.